### PR TITLE
fix(cert test): bump cert test deps to match main go mod ones

### PR DESCRIPTION
**Pull request by sicoyle**
# Description

I keep seeing an issue in my other PR bc of these dependencies. Want to separate the fix to here.
https://github.com/dapr/components-contrib/pull/3324
```
rror: /home/runner/go/pkg/mod/github.com/dapr/dapr@v1.13.2/pkg/runtime/hotreload/loader/disk/resource.go:33:21: not enough type arguments for type Batcher: have 1, want 2
Error: /home/runner/go/pkg/mod/github.com/dapr/dapr@v1.13.2/pkg/runtime/hotreload/loader/disk/resource.go:48:31: cannot infer T (/home/runner/go/pkg/mod/github.com/dapr/kit@v0.13.1-0.20240416202840-ccffb60016c7/events/batcher/batcher.go:50:24)
```

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #_[issue number]_

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [ ] Code compiles correctly
* [ ] Created/updated tests
* [ ] Extended the documentation / Created issue in the https://github.com/dapr/docs/ repo: dapr/docs#_[issue number]_

### CODE REVIEW

**Comment by berndverst** [comment]
You have to run `make modtidy-all` for linter and build to pass.

File: 3346.patch
Comment (berndverst) [inline 3346.patch:1]: I don't think this patch file should be part of this PR :)

**Comment by berndverst** [comment]
Any update here?

**Comment by sicoyle** [comment]
dropping in favor of https://github.com/dapr/components-contrib/pull/3390

**Pull request closed without merging**